Gemfile

```diff
@@ -1,8 +1,11 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'actionview'
 gem 'activerecord'
 
+gem 'nokogiri', '~> 1.6.0' if RUBY_VERSION.start_with?('2.0')
```

Comment:
👍 Let's leave this here for now in that case. Once we re-evaluate the versions we support, we can 💥 it.